kolibri/plugins/document_pdf_render/assets/src/module.js

```diff
@@ -1,16 +1,15 @@
 
 const KolibriModule = require('kolibri_module');
+const PDFComponent = require('./vue/index');
+const Vue = require('vue');
 
 class DocumentPDFModule extends KolibriModule {
-
-  render(contentData, renderArea) {
-    const documentTag = document.createElement('a');
-    documentTag.href = 'http://www.benlandis.com/sheet-music/piano-rickroll.pdf';
-    documentTag.textContent = contentData.id;
-    const titleTag = document.createElement('h1');
-    titleTag.innerHTML = contentData.id;
-    renderArea.appendChild(titleTag);
-    renderArea.appendChild(documentTag);
+  render(containerElement) {
+    const options = {
+      el: containerElement,
```

Comment:
@indirectlylit This is my main concern with this code as it currently stands - I am passing in a DOM element from the content-render card, but I feel like it is a very unVue way of doing it, as it doesn't put itself into the node hierarchy like instantiating a component normally would.

Is there a way I could pass around a slot instead and fill it dynamically?